tests/integration/cockroachdb_test.go

```diff
@@ -117,7 +117,7 @@ func (suite *CockroachDBSuite) TestCockroachDBClusterInstallation() {
 	command := "/cockroach/cockroach"
 	commandArgs := []string{
 		"sql", "--insecure", fmt.Sprintf("--host=cockroachdb-public.%s", suite.namespace), "-e",
-		`create database rookcockroachdb; use rookcockroachdb; create table testtable ( testID int ); insert into testtable values (123456789); select * from testtable;`,
+		`create database if not exists rookcockroachdb; use rookcockroachdb; create table if not exists testtable ( testID int ); insert into testtable values (123456789); select * from testtable;`,
```

Comment:
if this command executes twice, there will be no problem with the second insert because there are no unique constraint on the table, right?  And the verification later on will correctly find the value, even if it's duplicated.

did you get a chance to run this command multiple times manually to verify it's correct?  it looks correct, but just curious if you were able to fully confirm it.

Reply:
No, I didn't twice since I presumed it's correct.

Reply:
My head SQL interpreter also says this should work. 🙂